scene/3d/physics_body.cpp

```diff
@@ -92,17 +136,42 @@ void PhysicsBody::remove_collision_exception_with(Node* p_node) {
 	PhysicsServer::get_singleton()->body_remove_collision_exception(get_rid(),physics_body->get_rid());
 }
 
+void PhysicsBody::_set_layers(uint32_t p_mask) {
```

Comment:
Should there be a ``printline`` to notify users that this function is deprecated, and that they should use ``set_layer_mask`` and ``set_collision_mask`` directly?